src/protocols/rdp/rdp_settings.c

```diff
@@ -726,6 +732,11 @@ guac_rdp_settings* guac_rdp_parse_args(guac_user* user,
         guac_user_parse_args_boolean(user, GUAC_RDP_CLIENT_ARGS, argv,
                 IDX_ENABLE_PRINTING, 0);
 
+    /* Name of redirected printer */
+    settings->printer_name =
+        guac_user_parse_args_string(user, GUAC_RDP_CLIENT_ARGS, argv,
```

Comment:
This will need to be freed within `guac_rdp_settings_free()`.

Reply:
Ah, yes.

Reply:
Freed.